ddtrace/propagation/http.py

```diff
@@ -921,6 +921,10 @@ def _resolve_contexts(contexts, styles_w_ctx, normalized_headers):
                 ts = _extract_header_value(_POSSIBLE_HTTP_HEADER_TRACESTATE, normalized_headers)
                 if ts:
                     primary_context._meta[W3C_TRACESTATE_KEY] = ts
+                # W3C Phase 3: Ensure the last datadog parent id is always set on the primary context
```

Comment:
I am not sure "Phase 3" has context here in the comment, does it?

Reply:
Not really. It just makes sense to me. I will remove it